roles/grafana_cloud_operator/tasks/modify_alertmanager_secret.yml

```diff
@@ -23,6 +23,10 @@
           }
       }, recursive=True) }}
 
+- name: Print Provision Mode
+  ansible.builtin.debug:
+    msg: "Ansible Version: {{ provision_mode }}"
```

Comment:
I think this was added only for debugging purpose right? Remove if not required.

Reply:
PR Updated